### DISCUSSION TITLE
🔗Full control for normal, hover and disabled modes (Navigation arrows) - for Loop Grid and Nested Carousel (duplicate of #21235)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21290] [Feature Request]
### Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team @avivu , @nicholaszein.

Currently, the arrows in the Loop Grid widget have very limited customization. So, it would be great if you provide full control to customize them.

### Describe the solution you'd like

Add this possibility so that we can control arrows in 3 states. **normal**, **hover** and **disabled**

![NHD](https://user-images.githubusercontent.com/88034902/218253678-ad742add-b0c1-49a1-bbc0-16960e6173fb.png)

**The inactive state means that the carousel has reached its end. Currently, Elementor carousels have a disabled mode, but it is not customizable.**

The image below is something that would be great (image from the jet woobuilder plugin)

![preview](https://user-images.githubusercontent.com/88034902/218253929-cb93fe1e-75bd-4bf5-b63a-b2a51106453b.png)

Options needed to customize arrows in normal, hover and disabled modes :

- arrows icon size
- arrows icon color
- arrows background color
- arrows box height
- arrows box width
- border
- border radius
- box shadow

**Please keep in mind that these options are very much needed for a professional carousel.**

thanks

**Comment by nicholaszein** [comment]
duplicate of #21235
